### PR TITLE
[release notes] mention httpd/apache reverse proxy to re-enable SSL, add libevent dependency to build-osx.md

### PR DESCRIPTION
**Pull request by jonasschnelli**
^ see title.

### CODE REVIEW

File: doc/release-notes.md [hunk 1]
```diff
@@ -27,6 +27,38 @@ Then, to tunnel a SSL connection on 28332 to a RPC server bound on localhost on
 
 It can also be set up system-wide in inetd style.
 
+Another way to re-attain SSL would be to setup an httpd reverse proxy. This solution
```
Comment (fanquake) [inline doc/release-notes.md]: s/an/a

File: doc/release-notes.md [hunk 2]
```diff
@@ -27,6 +27,38 @@ Then, to tunnel a SSL connection on 28332 to a RPC server bound on localhost on
...
+would allow to use different authentication, loadbalancing, on-thy-fly compressing and
```
Comment (fanquake) [inline doc/release-notes.md]: s/allow to use/allow the use of

File: doc/release-notes.md [hunk 3]
```diff
@@ -27,6 +27,38 @@ Then, to tunnel a SSL connection on 28332 to a RPC server bound on localhost on
...
+    SSLCertificateFile /etc/apache2/ssl/file.pem
+
+    <Location /bitcoinrpc>
+        ProxyPass http://127.0.0.1:8332/
+        ProxyPassReverse http://127.0.0.1:8332/
+        # optional enable digest auth
+        # AuthType Digest
+        # ...
+        
+        # optioal bypass bitcoind rpc basic auth
```
Comment (fanquake) [inline doc/release-notes.md]: s/optioal/optional

**Comment by jonasschnelli** [comment]
Fixed @fanquake findings, added `libevent` dependency to `build-osx.md` (https://github.com/bitcoin/bitcoin/pull/5677#issuecomment-137736157)

**Comment by jgarzik** [comment]
ACK

**Comment by paveljanik** [comment]
And thanks for libevent :-)

**Comment by fanquake** [comment]
ACK

File: doc/release-notes.md [hunk 4]
```diff
@@ -27,6 +27,36 @@ Then, to tunnel a SSL connection on 28332 to a RPC server bound on localhost on
 
 It can also be set up system-wide in inetd style.
 
+Another way to re-attain SSL would be to setup a httpd reverse proxy. This solution
+would allow the use of different authentication, loadbalancing, on-thy-fly compressing and
```
Comment (paveljanik) [inline doc/release-notes.md]: compressing -> compression? But I'm not English native speaker...

**Comment by paveljanik** [comment]
ACK anyway. Thanks!

File: doc/release-notes.md (hunk 4 above)
Comment (laanwj) [inline doc/release-notes.md]: s/on-thy-fly/on-the-fly :)

Comment (jonasschnelli) [inline doc/release-notes.md]: Fixed compressing -> compression and s/on-thy-fly/on-the-fly.
Thanks.

**Pull request merged**